platform-api/src/routers/forms.py

```
@@ -0,0 +1,123 @@
+        raise HTTPException(400, "Turnstile verification failed")
+
+    # Create Sanity draft
+    doc_id = await create_submission(body, settings, sanity)
+
+    # Discord notification (fire-and-forget)
+    # FIXME currently this has been removed, maybe replaced with settings.kv.get("discord-webhook:{channel}") from 12.6
+    webhook_url = settings.optional_secrets.get("discord_webhook_url") 
+    try:
+        await notify_discord(body, webhook_url)
+    except Exception:
+        pass  # Don't block submission on Discord failure
```

Ruff (0.15.11)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)